src/signals/incident-management/containers/IncidentSplitContainer/IncidentSplitContainer.js

```diff
@@ -62,13 +62,13 @@ const IncidentSplitContainer = ({ FormComponent }) => {
   }, [getParent, id]);
 
   useEffect(() => {
-    if (isLoadingParent || isSuccessParent || errorParent === undefined) return;
+    if (errorParent === undefined && dataParent === undefined) return;
```

Comment:
I'm not a fan of all this `undefined` stuff and distinguishing between different types (`undefined`, `false`). We should do something about the `useFetch` implementation.